packages/core/src/agent/base.ts

```diff
@@ -287,6 +287,12 @@ export abstract class AgentRunner<
     return task.context.toolCallCount < MAX_TOOL_CALLS;
   }
 
+  protected computedMessageQuery = (
+    message: MessageContent,
+  ): MessageContent => {
+    return extractText(message);
+  };
```

Comment:
This is not necessary since each LLM already does `extractText` if not supported